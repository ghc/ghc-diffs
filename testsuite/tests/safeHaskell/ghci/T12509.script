import System.IO.Unsafe
import System.IO
