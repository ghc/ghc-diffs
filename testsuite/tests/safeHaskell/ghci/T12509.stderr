
<no location info>: error:
    System.IO.Unsafe: Can't be safely imported!
    The module itself isn't safe.

